python_modules/dagster/dagster/core/debug.py

```diff
@@ -27,10 +27,10 @@ def __new__(
             version=check.str_param(version, "version"),
             pipeline_run=check.inst_param(pipeline_run, "pipeline_run", PipelineRun),
             event_list=check.list_param(event_list, "event_list", EventLogEntry),
-            pipeline_snapshot=check.inst_param(
+            pipeline_snapshot=check.opt_inst_param(
                 pipeline_snapshot, "pipeline_snapshot", PipelineSnapshot
             ),
-            execution_plan_snapshot=check.inst_param(
+            execution_plan_snapshot=check.opt_inst_param(
```

Comment:
Can you load the debug dump without the snapshots?

Reply:
i'll just take this out for now, kind of a tangent from the change